src/components/icon/icon.spec.js

```diff
@@ -587,6 +588,21 @@ describe('MdIcon service', function() {
 
         $scope.$digest();
       });
+
+      it('should suffix duplicated ids and refs', function() {
+          // Just request the icon to be stored in the cache.
```

Comment:
Can you please update this code to use 2 space indentation (like the rest of the file) instead of 4 spaces?

Reply:
yes